pkg/ansible/flags/flag.go

```diff
@@ -28,6 +29,7 @@ type Flags struct {
 	ReconcilePeriod        time.Duration
 	WatchesFile            string
 	InjectOwnerRef         bool
+	MaxReconcile           int
```

Comment:
For clarity's sake (and even though it's a bit of a mouthful), I would suggest making the variable name `MaxConcurrentReconciles`
```suggestion
	MaxConcurrentReconciles int
```

Ditto for other variables like this one.